### ISSUE TITLE
setCurrentTripDestination

### ISSUE THREAD

**Issue by TyrantGummiBear** [ChasVanDav/budgetblog#15]
setCurrentTripDestination is never used which means currentTripDestination will always be null. You should consider what function/component would update or change the current trip and pass the setCurrentTripDestination to it so state can be updated.

**Comment by ChasVanDav** [comment]
Yes! I left this to do later and never got to it. Now that I am deployed in Render, I can shift my attention back to this very important gap! I will work on setting up this route. I am thinking this can be updated in an addTrip component where the user can input their "current destination."
